CONTRIBUTING.md

```diff
@@ -1,79 +1,106 @@
-# Contributing Guidelines
+# Contributing to Viewer
 
-## Setup
+Thank you for your interest in participating!
 
-Requires `node >= 12`.
+There are many ways to contribute, beyond writing code or programming, by: logging bugs, reporting issues, and creating suggestions. To do so, please [create a ticket](https://github.com/shabados/presenter/issues/new) in our issue tracker. See other ways to [Contribute](README.md#Contributing) or give [Feedback](README.md#Feedback).
 
-You will need to create a Google Cloud Project with the Translate API enabled to try out translation translating.
+This document is for developers or programmers contributing to the source code of Viewer.
 
-Translations will be enabled if the `GOOGLE_APPLICATION_CREDENTIALS` environment variable is set to a path that contains the credentials for a Google Cloud Project's service account, which you must create. See the [following steps](https://www.npmjs.com/package/@google-cloud/translate#quickstart) for further guidance on setting up the Google Cloud Project.
```

Comment:
These lines should be added back

Reply:
They're not deleted

https://github.com/shabados/viewer/blob/d2f0d8884bdd7e3f4973bf433be9a24e2eda374e/CONTRIBUTING.md#notes